data/dataAccess.go

```diff
@@ -1,4 +1,11 @@
 package data
 
 type DataAccess interface {
-}
+	// Not entirely sure what to the return type is going to be here
+	CreateUser([]string) (result string, err error)
```

Comment:
Its not typical to specify the name of the return results in go although you can do it. I would recommend (string, err)

Reply:
Sounds good